python_modules/dagster/dagster/_daemon/monitoring/run_monitoring.py

```diff
@@ -156,7 +156,9 @@ def monitor_started_run(
                 instance.report_run_failed(run, msg)
                 # Return rather than immediately checking for a timeout, since we just failed
                 return
-    check_run_timeout(instance, run_record, logger)
+    check_run_timeout(
+        instance, run_record, logger, float(instance.run_monitoring_max_runtime_seconds)
```

Comment:
Why does this have to be cast? Doesn't our config already make sure we're passing through an int?